interface/wx/string.h

```diff
@@ -743,13 +743,16 @@ class wxString
         (underscore) character.
 
         Use mb_str() or utf8_str() to convert to other encodings.
+        
+        @param replaceWith
+            The Character that will replace found unicode character. defaults to underscore "_"
```

Comment:
Could you please lower case "Character", upper case "defaults" and add "This parameter is new since wxWidgets 3.1.0."?